### PR TITLE
Largest subarray sum

### PR DESCRIPTION
**Pull request by anubhav-sharma13**
It contains the code of how to find the largest subarray sum in Python.

### CODE REVIEW

**Comment by cclauss** [comment]
Travis CI tests are failing.  See CONTRIBUTING.md for details.

This repo no longer supports legacy Python.  `print()` is a function in Python 3.

https://travis-ci.com/TheAlgorithms/Python/builds/132670589#L352

File: other/largest_subarray_sum.py [hunk 1]
```diff
@@ -15,4 +15,4 @@ def maxSubArraySum(a, size):
    
 # Driver function to check the above function  
 a = [-13, -3, -25, -20, -3, -16, -23, -12, -5, -22, -15, -4, -7] 
-print "Maximum contiguous sum is", maxSubArraySum(a, len(a)) 
+print ("Maximum contiguous sum is", maxSubArraySum(a, len(a)))
```
Comment (cclauss) [inline other/largest_subarray_sum.py]: No space between print and (

File: sorts/i_sort.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  
+        key = arr[i] 
+        j = i-1
+        while j >=0 and key < arr[j] : 
+                arr[j+1] = arr[j] 
+                j -= 1
+        arr[j+1] = key 
+arr = [12, 11, 13, 5, 6] 
+insertionSort(arr) 
+print ("Sorted array is:")
```
Comment (cclauss) [inline sorts/i_sort.py]: No space between print and (

File: sorts/i_sort.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        j = i-1
+        while j >=0 and key < arr[j] : 
+                arr[j+1] = arr[j] 
+                j -= 1
+        arr[j+1] = key 
+arr = [12, 11, 13, 5, 6] 
+insertionSort(arr) 
+print ("Sorted array is:") 
+for i in range(len(arr)): 
+    print ("%d" %arr[i])
```
Comment (cclauss) [inline sorts/i_sort.py]: No space between print and (

**Comment by cclauss** [comment]
`sys.maxint` was removed in Python 3 because all integers are of infinite precision.
https://docs.python.org/3/whatsnew/3.0.html#integers

**Comment by cclauss** [comment]
This is what a Pythonic function name, type hints, a doctest, and a default argument would look like.

```python
def max_sub_array_sum(a: list, size: int = 0):
    """
    >>> max_sub_array_sum([-13, -3, -25, -20, -3, -16, -23, -12, -5, -22, -15, -4, -7])
    -3
    """
    size = size or len(a)
```

**Comment by cclauss** [comment]
https://travis-ci.com/TheAlgorithms/Python/builds/132847594#L544

**Comment by anubhav-sharma13** [comment]
It is working on my system but the travis CI test failed

**Comment by cclauss** [comment]
On your system, what happens when you type:
* python3 -m doctest largest_subarray_sum.py
* python3 -m doctest -v largest_subarray_sum.py

**Comment by cclauss** [comment]
Doctests are just docstrings until you run `python3 -m doctest` on them and then they turn into tests!

File: other/largest_subarray_sum.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 from sys import maxsize 
-def maxSubArraySum(a: list, size: int = 0):
+def max_Sub_Array_Sum(a: list, size: int = 0):
     """
     >>> max_sub_array_sum([-13, -3, -25, -20, -3, -16, -23, -12, -5, -22, -15, -4, -7])
```
Comment (cclauss) [inline other/largest_subarray_sum.py]: Let’s be Pythonic and use `max_sub_array_sum` everywhere.

**Pull request merged**